### DISCUSSION TITLE
Allow changing font size app wide/making the design more compavt

### DISCUSSION THREAD

**Discussion by uvesten** [community/community#4204] [Mobile]
Honestly, I'm more productive using github.com than with the app on my iphone 8 plus. 
I don't have as much information per screen in the app.
Could you please add an option to minimize font/ make layout more compact?

**Comment by amrehman** [comment]
Hi @uvesten thanks a lot for getting in touch!
That is a great idea, which we would like to explore!

Would you like to provide an example where you find your productivity is hindered in the app and having a more compact layout will make things much better for you?
